proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -109,15 +109,34 @@ message ReadDataResponse {
   ChainPoint ledger_tip = 2; // The chain point that represent the ledger current position.
 }
 
+// Request to get a transaction by hash
+message GetTxRequest {
+  bytes hash = 1; // The hash of the transaction.
+  google.protobuf.FieldMask field_mask = 2; // Field mask to selectively return fields in the response.
+}
+
+// Represents a transaction from any supported blockchain.
+message AnyChainTx {
+  oneof chain {
+    utxorpc.v1alpha.cardano.Tx cardano = 1; // A Cardano transaction.
+  }
+}
+
+// Response containing the transaction associated with the requested hash.
+message GetTxResponse {
+  AnyChainTx tx = 1; // The transaction.
+  ChainPoint ledger_tip = 2; // The chain point that represent the ledger current position.
+}
+
 // Service definition for querying the state of the chain.
 service QueryService {
   rpc ReadParams(ReadParamsRequest) returns (ReadParamsResponse); // Get overall chain state.
   rpc ReadUtxos(ReadUtxosRequest) returns (ReadUtxosResponse); // Read specific UTxOs by reference.
   rpc SearchUtxos(SearchUtxosRequest) returns (SearchUtxosResponse); // Search for UTxO based on a pattern.
   rpc ReadData(ReadDataRequest) returns (ReadDataResponse); // Read specific datum by hash
+  rpc ReadTxs(GetTxRequest) returns (GetTxResponse); // Get Txs by by chain-specific criteria.
```

Comment:
Since this method works by requesting one tx at a time, I would call it `ReadTx`.